core/src/test/scala/shapeless/serialization.scala

```diff
@@ -228,7 +234,7 @@ object SerializationTestDefns {
 
 }
 
-class SerializationTests {
+class SerializationTests extends Serializable {
   import SerializationTestDefns._
```

Comment:
Some of its changes got lost, yes (around [this line](https://github.com/milessabin/shapeless/commit/78def231b91b45786aa9676fc1398b868056ac04#diff-ffd1855210ab80b59329091ae5193e48R194)), but don't fix all of the issues here. I still have to comment out [two tests](https://github.com/milessabin/shapeless/commit/720164d1658354b32ec7725243b642665536d900#diff-21cd9467b2b35a579fc7910defb1d163R1085) (reverting the changes I made to them doesn't solve the issue either).